### DISCUSSION TITLE
Save data to another DataSource

### DISCUSSION THREAD

**Discussion by YoonSungHyun-Git** [cybertec-postgresql/pgwatch2#536] [Q&A]
Hello,

I want to store the collected data in different datasources. Is there a way?

For example 

MetricName : storeInflux
MetricName : storePostgresql

If there are two metric SQLs, I want to store one as influx and one as postgres.

If you have any relevant information, please share it.

Thanks

Yoon.

**Comment by pashagolub** [comment]
Hello, 

straightforward solution would be to use 2 pgwatch instances. One storing data into Influx another to PostgreSQL

**Reply by YoonSungHyun-Git** [reply]
Hello,
With a lot of monitoring and the amount of data to store, we are looking for a more efficient way. Is there any other alternative? Among the collector sources, according to the Datastore variable, it is divided into influx and postgres collection, can we not use this variable?
Can't we use Metric Attr in config to branch options into structs?
I would also like to know how to use the Option structure. If you have any references or links, please do.

![image](https://user-images.githubusercontent.com/95339676/203878214-069fec19-c396-4300-a55b-3fdf2e7dd94b.png)

![image](https://user-images.githubusercontent.com/95339676/203878275-66a8349d-2c57-42c0-95fd-ee8cacdea959.png)

**Reply by pashagolub** [reply]
Hello, I'm not aware of any alternatives. 

But since this is the open source project, pull requests are more then welcome.
